server/src/routes/newRelicErrors.js

```diff
@@ -26,21 +27,27 @@ const router = express.Router();
 router.get('/', (req, res, next) => {
   const nrql = 'SELECT count(*) FROM TransactionError WHERE appName = \'www.campsy.de\' SINCE 7 DAYS AGO COMPARE WITH 1 week ago';
 
+  const cacheKey = req.baseUrl;
+  const cacheService = req.app.locals.services.cacheService;
+  const cachedPayload = cacheService.get(cacheKey);
+
   // TODO: Refactor to use Promise;
-  insights.query(nrql, (err, responseBody) => {
-    if (err) {
-      next(err);
-      return;
-    }
-    res.json({
-      status: 'success',
-      message: '',
-      data: {
-        lastWeek: _get(responseBody, 'previous.results[0].count'),
-        thisWeek: _get(responseBody, 'current.results[0].count'),
-      },
+  if (cachedPayload) {
+    res.json(getResponseSuccess(cachedPayload));
+  } else {
+    insights.query(nrql, (err, insightsResponse) => {
+      if (err) {
+        next(err);
+        return;
+      }
+      const payload = {
+        previous: _get(insightsResponse, 'previous.results[0].count'),
+        current: _get(insightsResponse, 'current.results[0].count'),
+      };
+      res.json(getResponseSuccess(payload));
+      cacheService.set(cacheKey, payload);
```

Comment:
I'm not sure how much important it here. I think the positions of the last two lines should be switched.